package.json

```diff
@@ -81,7 +81,7 @@
     "gl-mat4": "^1.2.0",
     "gl-mesh3d": "^2.1.1",
     "gl-plot2d": "^1.4.2",
-    "gl-plot3d": "^2.2.2",
+    "gl-plot3d": "git://github.com/gl-vis/gl-plot3d.git#4d41ca21b3c0a25db8776ce92261995323bb62e8",
```

Comment:
Have you made a PR for this patch?

Reply:
https://github.com/gl-vis/gl-plot3d/pull/24
@etpinard Thanks!